File: v2/snippets/testResources/nodejs-package/TestClass.js
class TestClass {

    publicValue;
    #privateValue;

    constructor(first, second) {
        this.publicValue = first;
        this.#privateValue = second;
    }

    static multiplyByTwo(value) {
        return value * 2;
    }

    static staticValue = 3;

    multiplyTwoNumbers(first, second) {
        return first * second
    }
}

module.exports = TestClass

File: v2/snippets/testResources/ruby-package/TestClass.rb
module TestClass
  class TestClass
    @@static_value = 3
    @public_value

    private
    @private_value

    def initialize(public_value, private_value)
      @public_value = public_value
      @private_value = private_value
    end

    def self.multiply_by_two(value)
      return value * 2
    end

    def multiply_two_numbers(first, second)
      return first * second
    end
  end
end